source/widgets/UnixWindow.ooc

```diff
@@ -43,5 +44,15 @@ UnixWindow: class extends DisplayWindow {
 	refresh: override func {
 		this _openGLWindow refresh()
 	}
+	processEvents: override func {
+		while (XPending(this _xWindow display)) {
+			event: XEventOoc
+			XNextEvent(this _xWindow display, event&)
+			if ((event type == ButtonPress) && this _mousePressHandler)
+				this _mousePressHandler call(IntPoint2D new(event xkey x, event xkey y))
+			if ((event type == ButtonRelease) && this _mouseReleaseHandler)
+				this _mouseReleaseHandler call(IntPoint2D new(event xkey x, event xkey y))
+		}
+	}
```

Comment:
You are right, but this will be gone from both the `UnixWindow` and `X11Window`, as it in fact belongs to the, yet non-exisiting, `X11Application` class, as events processing can happen even without the window dispplayed on screen. I want to extend this soon, this is only a temporary solution for Bettina's problem in the other project.